Packs/ExabeamDataLake/Integrations/ExabeamDataLake/ExabeamDataLake.yml

```diff
@@ -0,0 +1,76 @@
+category: Analytics & SIEM
+commonfields:
+  id: Exabeam Data Lake
+  version: -1
+configuration:
+- defaultvalue: https://example.com/
+  display: Server URL
+  name: url
+  required: true
+  type: 0
+- display: User Name
+  name: credentials
+  required: true
+  section: Connect
+  type: 9
+  displaypassword: Password
+- defaultvalue: local
+  display: Cluster Name
+  name: cluster_name
+  type: 0
+  required: true
+  additionalinfo: The default value is usually 'local', suitable for standard setups. For custom cluster deployments, consult Exabeam Support Team.
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+description: Cloud-delivered Data Lake provides a highly scalable, cost-effective, and searchable log management system. Data Lake is used for log collection, storage, processing, and presentation.
+display: Exabeam Data Lake
+name: Exabeam Data Lake
+script:
+  commands:
+  - arguments:
+    - description: The search query string to filter the events by. (e.g. "VPN". See more examples in the integration description).
+      name: query
+      required: true
+    - defaultValue: "7 days ago"
+      description: The starting date for the search range. The search range should be at least one day long and can extend up to a maximum of 10 days.
+      name: start_time
+    - defaultValue: "today"
+      description: The ending date for the search range. This defines the end of the search range, which should be within one to ten days after the start_time.
+      name: end_time
+    - description: The maximal number of results to return. Maximum value is 3000.
+      name: limit
+    - description: The page number for pagination.
+      name: page
+    - description: The maximal number of results to return. Maximum value is 3000.
```

Comment:
Is this description correct?
Should it be: The maximum number of results to return per page.
Is the maximum value correct?

Reply:
yes